generate_template.sh

```diff
@@ -16,18 +16,27 @@ then
      rm .compute.properties
 fi
 
+mkdir -p ${WORKDIR}/generatedscripts/
```

Comment:
Deze bestaat allang, is overbodig lijkt me.. Waar laat je dit script eigenlijk, want als je nu nog een generatedscripts wilt maken, waar run je dit script?